src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -182,17 +207,17 @@ const AnalysisCard = ({
 
   //the flex values for columns here correspond to the flex values in the header
   const artifactName = div({
+    onClick: () => Nav.goToPath(analysisLauncherTabName, { namespace, name: workspaceName, analysisName }),
     title: getFileName(name),
-
     role: 'cell',
     style: {
-      ...Style.elements.card.title, whiteSpace: 'normal', overflowY: 'auto', textAlign: 'left', ...centerColumnFlex
+      ...Style.elements.card.title, whiteSpace: 'normal', textAlign: 'left', cursor: 'pointer', ...centerColumnFlex
     }
   }, [
-    a({ href: analysisLink }, [getFileName(name)])
```

Comment:
This looks like it gets rid of a link element and replaces it with a clickable div? How does this impact accessibility? Can this still be used with a keyboard?

Reply:
Yes, tested with axe dev tools. This was actually a bug introduced when we made accessibility changes, essentially made only the text on the banner clickable (now the whole banner is clickable)